src/index.html

```diff
@@ -7,8 +7,49 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="image"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="#" class="nav-link is-active">Apple</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">Samsung</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav-item">
+            <a
+              href="#"
+              class="nav-link"
+              data-qa="hover"
+            >Ноутбуки и компьютеры</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">Гаджеты</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">Планшеты</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">Фото</a>
+          </li>
+          <li class="nav-item">
```

Comment:
you don't have any rules for `.nav-item`, so this class is useless 😉

Reply:
fixed :-)